flink-core/src/main/java/org/apache/flink/api/java/typeutils/TypeExtractor.java

```diff
@@ -1723,20 +1720,17 @@ private boolean isValidPojoField(Field f, Class<?> clazz, ArrayList<Type> typeHi
 					// return type is void.
 					m.getReturnType().equals(Void.TYPE)
 				) {
-					if(hasSetter) {
-						throw new IllegalStateException("Detected more than one setter");
-					}
 					hasSetter = true;
 				}
 			}
 			if(hasGetter && hasSetter) {
 				return true;
 			} else {
 				if(!hasGetter) {
-					LOG.debug(clazz+" does not contain a getter for field "+f.getName() );
+					LOG.info(clazz+" does not contain a getter for field "+f.getName() );
```

Comment:
Does it make sense to log this on `info`? That can be a lot of log output for programs with many types...